state/syncer/userAccountsSyncer.go

```diff
@@ -274,8 +274,8 @@ func (u *userAccountsSyncer) syncAccountDataTries(
 }
 
 func (u *userAccountsSyncer) printDataTrieStatistics() {
-	u.mutStatistics.RLock()
-	defer u.mutStatistics.RUnlock()
+	u.mutStatistics.Lock()
```

Comment:
Why this change?

Reply:
because there is a `sort.Slice(u.largeTries, func(i, j int) bool ` that happened under RLock/RUnlock. It's not really an issue since the `printDataTrieStatistics` function is called sequentially, but it's better to have the fix

Reply:
ok